### PR TITLE
fix: pass useNativeDriver to all animations

### PR DESCRIPTION
**Pull request by Trancever**
### Motivation

Running animations without explicitly passing `useNativeDriver` property on React-Native > 0.60 is causing warnings.

### Test plan

Run the example app and check if animations work as expected.

### CODE REVIEW

File: src/components/Banner.tsx [hunk 1]
```diff
@@ -151,6 +151,7 @@ class Banner extends React.Component<Props, State> {
     Animated.timing(this.state.position, {
       duration: 250 * scale,
       toValue: 1,
+      useNativeDriver: false,
```
Comment (jayu) [inline src/components/Banner.tsx:154]: If this doesn't work with the native drivers, you may try storing height as an animated value. This is only a suggestion, but it feels that banner position is animated in JS

Comment (Trancever) [reply src/components/Banner.tsx:154]: Yes it is, we were not able to achieve the desired behaviour with animating translation only.

**Comment by kabus202** [comment]
Can you also include CheckboxAndroid.tsx?

**Comment by Trancever** [comment]
@kabus202 Yeah, will do it. Thanks for pointing it!

**Pull request merged**